packages/sdk/src/instant-trade/InstantTradeSellerTxBuilder.ts

```diff
@@ -75,10 +77,10 @@ export default class InstantTradeSellerTxBuilder extends InstantTradeBuilder {
     }
     const amount = Math.ceil(royalty.pct * this.price)
 
-    return {
-      address: royalty.address as string,
-      amount: amount >= MINIMUM_AMOUNT_IN_SATS ? amount : 0
-    }
+    this.setRoyalty({
+      amount,
+      receiver: royalty.address as string
+    })
```

Comment:
we have `royalty.pct` here why not pass it to `setRoyalty` too?

Reply:
Refactored; https://github.com/sadoprotocol/ordit-sdk/pull/85/commits/ff017eeb3b84ef48be865df7e008746220e3bbc9